Sources/ArcGISToolkit/Components/FormView/FormViewModel.swift

```diff
@@ -88,10 +84,8 @@ import SwiftUI
         isEvaluating = true
         evaluateTask = Task {
```

Comment:
Should this be a detached task? Otherwise the task is implicitly holding onto self strongly, preventing it from the model from going away and the task from being cancelled.